README.md

```diff
@@ -13,11 +13,20 @@ transactions if you require them. Scale horizontally with a few mouse clicks.
 The supported data models can be mixed in queries and allow ArangoDB to be the
 aggregation point for your data.
 
-To get started, try one of our 10 minutes [tutorials](https://www.arangodb.com/tutorials)
-in your favorite programming language or try one of our [ArangoDB Cookbook recipes](https://www.arangodb.com/docs/stable/cookbook/).
+Check out our [training center](https://www.arangodb.com/arangodb-training-center/)
+to get started and see the full [documentation](https://www.arangodb.com/docs/stable/)
+to dive deeper.
 
-For the impatient: [download](https://www.arangodb.com/download) and install
-ArangoDB. Start the server `arangod` and point your browser to `http://127.0.0.1:8529/`.
+For the impatient:
+
+- Start the ArangoDB Docker container
+
+      docker run -e ARANGO_ROOT_PASSWORD=test123 -p 8529:8529 -d arangodb/arangodb
```

Comment:
That will pick `arangodb/arangodb:latest` official docker image? Are we aware of it in devel README.md?

Reply:
I would argue that it's a devel readme, but a general readme. If anyone comes across ArangoDB on GitHub, they should try the latest *stable* version and not a potentially broken devel version. I think this is fine as it is. Also, if it was a readme specifically for readme, we would need to change it for each new version branch we create from devel (not a fan, would likely be forgotten).